src/Knp/Rad/User/Salt/Generator/PseudoRandomBytesGenerator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Knp\Rad\User\Salt\Generator;
+
+use Knp\Rad\User\Salt\Generator;
+
+class PseudoRandomBytesGenerator implements Generator
+{
+    /**
+     * @var int
+     */
+    private $length;
+
+    /**
+     * @param int $length
+     */
+    public function __construct($length)
+    {
+        $this->length = $length;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function generate()
+    {
+        return openssl_random_pseudo_bytes($this->length);
```

Comment:
In fact it's not a parameter to as a strong encryption, it's a parameter by reference ans setted to true if the generated string has been generated by a strong encryption algorithm...